streams/core.py

```diff
@@ -7,6 +7,7 @@
 from tornado.ioloop import IOLoop
 from tornado.queues import Queue
 from collections import Iterable
+from streams.graph import visualize
```

Comment:
This comes from the dask api which had visualize as a method. But I don't have too much feeling either way. Although thinking about it now we may want the graph creation exposed as a method in case users are interested.